stdlib/uchar.mli

```diff
@@ -143,6 +143,20 @@ val utf_decode_invalid : int -> utf_decode
     smaller or equal to [4] (this is not checked by the module). The
     resulting decode has {!rep} as the decoded Unicode character. *)
 
+val utf_8_decode_length_of_byte : char -> int
+(** [utf_8_decode_length_of_byte byte] is the number of bytes that a
+    UTF-8 decode starting with byte [byte] would consume. This is either
+    [1], [2], [3] or [4] or [0] if [byte] is an invalid UTF-8 start
+    byte.
+
+    @since 5.4 *)
+
+val max_utf_8_decode_length : int
+(** [max_utf_8_decode_length] is [4], the maximal number of bytes
+    a valid or invalid UTF-8 decode can consume.
```

Comment:
If we have an explicit name available, I would propose to quote it directly in `utf_decode_length`, `utf_decode` and `utf_decode_invalid` rather than using `4` .

Reply:
The functions you mention are agnostic to the Unicode Transformation Format, here we talk about UTF-8.

Reply:
Good point.

Reply:
Also they are agnostic to the unit of elements (it was envisioned that `Utf.utf_decode` could also be used, for example to decode, a JavaScript string which have 16-bit code points as elements). Here this being UTF-8 we talk explicitely about *bytes*.